crates/types/src/network.rs

```diff
@@ -21,6 +22,7 @@ pub struct Network {
 impl Network {
     pub fn mainnet() -> Self {
         Self {
+            internal_id: Some(0),
```

Comment:
this isn't right. we can't assume this will have id 0. in fact it should probably have `None`, and be filled only when actually added to the persistent settings